src/cr/cube/cube.py

```diff
@@ -791,8 +855,10 @@ class _WeightedCountMeasure(_BaseMeasure):
     def _flat_values(self):
         """Optional 1D np.ndarray of np.float64 numeric counts after weighting."""
         unweighted_counts = self._cube_dict["result"]["counts"]
-        weighted_counts = self._cube_dict["result"]["measures"]["count"]["data"]
-        if unweighted_counts == weighted_counts:
+        weighted_counts = (
+            self._cube_dict["result"]["measures"].get("count", {}).get("data")
+        )
+        if unweighted_counts == weighted_counts or weighted_counts is None:
```

Comment:
This doesn't seem right. The presence of flat values for weighted-counts have nothing to do with unweighted counts. If a `result.measures.count` payload item appears, we return it here. Otherwise we return None. It containing equivalent values to `result.count` does not change the fact that these are present.

Basically, if `cube-count` appears in the query, then this property has a non-None value.

Reply:
This is always been this way. If you take a look on `master` it has the check on the unweighted counts either.

Reply:
Basically, if wcounts and ucounts are the same means that counts are not weighted and so this check i think

Reply:
Ok i investigate on that and the problem is that all the fixtures have `result.measures.count` so this means that all the cube have weighted counts. This is no longer optional and that's not true. So I'd prefer to keep it this way until we figure out what the best thing to do.